Integrate doc pushing for py.delta.chat (and c.delta.chat)

Discussed with @azul to help us with setting up travis secret management. Plan: 

make travis builds upload branches and master docs of "py" docs and maybe also "c" docs. 

- py docs are currently pushed with "cd python/doc ; make install" which syncs it to delta@codespeak.net:build/* and nginx on that codespeak server handles the ssl and serving the docs. 

- C docs ("cd docs ; doxygen") are currently served via github.io -- we can easily let "https://c.delta.chat" point to a directory under the delta account. 

azul, i added your ssh key so you can login and add a new (travis-push) key to authorized keys.

Looks like i am missing an dependency...
```
~/code/deltachat-core/python/doc$ make install
sphinx-build -b html -d _build/doctrees   . _build/html
Traceback (most recent call last):
  File "/home/azul/.local/bin/sphinx-build", line 7, in <module>
    from sphinx.cmd.build import main
ImportError: No module named sphinx.cmd.build
Makefile:86: recipe for target 'html' failed
make: *** [html] Error 1
```

On Tue, Sep 25, 2018 at 05:04 -0700, azul wrote:
> Looks like i am missing an dependency...
> ```
> ~/code/deltachat-core/python/doc$ make install
> sphinx-build -b html -d _build/doctrees   . _build/html
> Traceback (most recent call last):
>   File "/home/azul/.local/bin/sphinx-build", line 7, in <module>
>     from sphinx.cmd.build import main
> ImportError: No module named sphinx.cmd.build
> Makefile:86: recipe for target 'html' failed
> make: *** [html] Error 1
> ```

this looks odd -- could you reinstall sphinx,
e.g. "pip install --force-reinstall --user sphinx"?
(not quite sure about your setup -- you seem to 
be using "--user" directories -- i usually use
virtualenv's)

--- a/.scripts/deploy.sh
+++ b/.scripts/deploy.sh
@@ -27,8 +27,14 @@ printf "Host *\n" >> ~/.ssh/config
 printf " %sAuthentication no\n" ChallengeResponse Password KbdInteractive >> ~/.ssh/config
 
 
-# Perform the actual deploy
+# Perform the actual deploy to py.delta.chat
 rsync -avz \
   -e "ssh -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null" \
   $TRAVIS_BUILD_DIR/python/doc/_build/html/ \
   ${DEPLOY_USER}@${DEPLOY_SERVER}:build/${TRAVIS_BRANCH/\//_}
+
+# Perform the actual deploy to c.delta.chat
+rsync -avz \
+  -e "ssh -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null" \
+  $TRAVIS_BUILD_DIR/docs/html/ \
+  ${DEPLOY_USER}@${DEPLOY_SERVER}:build-c/${TRAVIS_BRANCH/\//_}

